### ISSUE TITLE
modal.setFrame()?

### ISSUE THREAD

**Issue by rcarmo** [kasper/phoenix#278]
I'm trying to draw window overlays and drag regions, and It seems I can't resize a Modal (which would do nicely for a transient full-window overlay) in any way, at least according to the docs.

Any workarounds?

**Comment by kasper** [comment]
Modals auto-resize. Currently the only way to adjust the size is to set the `weight` property of an modal.

**Comment by rcarmo** [comment]
Yes, I have come to realize that. I need to draw a window-sized rectangle and place it arbitrarily on screen, so I guess there is no way to do that in Phoenix currently.

**Comment by kasper** [comment]
You can arbitrarily place the modal anywhere on the screen, but yes if would want to have an exact size, that is not currently possible. I’ll think about that.
